codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ConstrainedCollectionGenerator.kt

```diff
@@ -121,23 +122,23 @@ class ConstrainedCollectionGenerator(
 
         writer.rustTemplate(
             """
-            impl #{TryFrom}<$inner> for $name {
-                type Error = #{ConstraintViolation};
+        impl #{TryFrom}<$inner> for $name {
```

Comment:
This looks like it needs to be reverted, the formatting was correct before (the beginning of `impl` should align with the first double quote in the preceding line).